roles/client_side_tests/tasks/get_prom_info.yml

```diff
@@ -1,4 +1,17 @@
 ---
+- name: "Get kube password"
+  ansible.builtin.shell:
+    cmd: |
```

Comment:
This is very specific to the installer used to provision openshift.

This won't work, for example, on a cluster created with CRC.

I'd prefer to keep the login part as a pre-task that the user is expected to perform before running the functional tests.